tests/integration-tests.js

```diff
@@ -214,11 +214,12 @@ describe( 'The database', () => {
 
   it( 'creates the ' + config.scoresTableName + ' table on the first request', async() => {
     expect.hasAssertions();
-    await points.updateScore( defaultItem, '++' );
+    const newScore = await points.updateScore( defaultItem, '+' );
```

Comment:
Hm. I don't think it actually matters here - we're testing integration with the database, not that the command works, so the result of this method or if it works _at all_ are mostly irrelevant.

That said, this isn't a bad thing, so :woman_shrugging:

Reply:
Actually this matters when using parameterized SQL because of the way the operation is used to construct "+1" or "-1" to pass into the parameter list. I am not sure how the constructed SQL by concatenation actually did not throw an exception as it would have been invalid SQL value syntax for integer.

The specific error was that the parameter `++1` is not a valid integer value. Since the production runtime is not making this mistake the error was benign but broke this unit test.

Reply:
A further comment: as with the test further on, this should be verifying that the database has been updated, not the return value, and I think that has already happened.